auctions/templates/addon/logon.html

```diff
@@ -0,0 +1,30 @@
+{% extends "addon/base.html" %}
+{% load socialaccount %}
+{% load staticfiles %}
+
+{% block widget_content %}
+    <div id="widget-wrapper">
+
+        <img id="widget-avatar" src="{{ user.get_gravatar_url }}" width="48" height="48"></img>
```

Comment:
Can remove this - we won't have it when the user sees this view because they're not logged in.